.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,26 @@
-^ Make sure to prefix the Title with [WIP]/[WIPTEST]
+<!-- Make sure to prefix the PR Title in the following flow...
 
-Purpose or Intent
-=================
+   [WIP] - Work in Progress
+   [WIPTEST] - Work in Progress with PRT Testing
+   [RFR] - Ready for 1st Review
+   [1LP][WIP] - 1st Level Pass, Work in Progress
+   [1LP][WIPTEST] - 1st Level Pass, Work in Progress with PRT Testing
+   [1LP][RFR] - 1st Level Pass, Ready for 2nd Review
 
-__Fixing__ X because it is currently doing Y which is incorrect. It should be doing Z.
+-->
 
-or
+## Purpose or Intent
+<!-- Please provide some information related to PR. Some examples are:
 
-__Extending__ X because I need it to do Y so that I can update / add more test cases; PR #123 depends on it.
+- __Fixing__ X because it is currently doing Y which is incorrect. It should be doing Z.
+- __Extending__ X because I need it to do Y so that I can update/add more test cases; PR #123 
+depends on it.
+- __Adding tests__ for feature X to cover cases Y and Z. They were implemented in product version 1.2.3.
+- __Updating tests__ for feature X because the behavior changed in product version 1.2.3.
+- __Enhancement__ for feature X
 
-or
+-->
 
-__Adding tests__ for feature X to cover cases Y and Z. They were implemented in product version 1.2.3.
-
-or
-
-__Updating tests__ for feature X because the behavior changed in product version 1.2.3.
-
---
-These should serve mainly as a guideline. The more related information you provide, the better. :wink:
+### PRT Run
+<!-- Try to run tests for each PR or prefix the PR Title with [NOTEST]-->
```

Comment:
Does appending [NOTEST] have any impact on PRT run ? I think it still runs tests.

Reply:
@sshveta It will still run the smoke tests AFAIK

Reply:
This is something that can be addressed above, by adding a line about [NOTEST]